tests/end2end/playwright/location_search.spec.js

```diff
@@ -0,0 +1,136 @@
+// @ts-check
+import { test, expect } from '@playwright/test';
+
+test.describe('Location search', () => {
+
+    test('Default', async ({ page }) => {
+        const url = '/index.php/view/map?repository=testsrepository&project=location_search';
+        // Wait for WMS GetCapabilities promise
+        let getCapabilitiesWMSPromise = page.waitForRequest(/SERVICE=WMS&REQUEST=GetCapabilities/);
+        await page.goto(url);
```

Comment:
```suggestion
        await page.goto(url);
```

Can you use `gotomap` ? (you must just add a single layer in the project for now, as you have noticed)